contracts/exchange/contracts/src/MixinWrapperFunctions.sol

```diff
@@ -142,7 +142,8 @@ contract MixinWrapperFunctions is
         return fillResults;
     }
 
-    /// @dev Executes multiple calls of fillOrderNoThrow until total amount of takerAsset is sold by taker.
+    /// @dev Executes multiple calls of fillOrder until total amount of takerAsset is sold by taker.
```

Comment:
We mentioned that it would be good to document the changes to the restrictions on `takerAssetData` and `makerAssetData`. Where will this be documented?

Reply:
In the spec (using multiple stablecoins as an example).

Reply:
Actually just decided to add a note for each of these functions too.